submissions/Nik3264/js-dom/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@400;700&family=Permanent+Marker&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header class="header">
+        <h1>Music that inspires...</h1>
+    </header>
+    <main class="wrap">
+        <nav class="menu">
+            <div class="menu__item">Led Zeppelin</div>
+            <div class="menu__item menu__item__active">Beatles</div>
+            <div class="menu__item">Deep Purple</div>
+            <div class="menu__item">Pink Floyd</div>
+            <div class="menu__item">Scorpions</div>
+        </nav>
+        <div class="content">
+            <div class="item">
+
```

Comment:
A redundant empty line. Configure your code editor to see redundant space/tabs/empty lines.

Reply:
Ok, I configured my code with Prettier.